examples-nosql-java-sdk/CreateLoadComplexTable/README.md

```diff
@@ -0,0 +1,347 @@
+# Overview
+
+The `CreateLoadComplexTable` example program creates a table in the Oracle NoSQL Database Cloud Service and then uses the [Oracle NoSQLDatabase Java SDK](https://docs.oracle.com/en-us/iaas/nosql-database/doc/oracle-nosql-database-java-sdk.html) to populate the table with sample records; where the table that is created consists of all the data types (primitive and complex) currently supported by the NoSQL Cloud Service. This example program is referenced in the [Oracle NoSQL Database Analytics Integrator User Guide](https://docs.oracle.com/en/cloud/paas/nosql-cloud/zsons/index.html#articletitle).
+ 
+The tasks you can perform with this example are:
+
+  - Modify and execute the source code that is provided.
+  - Modify and run the unit tests that are provided.
+  - Build a package for release that can then be installed and executed in the desired environment.
+
+The program can be executed in one of the three environments listed below. How the program authenticates 
+with the Oracle NoSQL Database Cloud Service depends on the environment, as well as how the program is configured (described below).
+
+  - The OCI Command Shell; where authentication is performed using either an OCI Delegation Token or the user's private credentials.
+  - An OCI Compute Instance; where authentication is performed using either an OCI Instance Principal or the user's private credentials.
+  - The user's local (non-cloud) environment; where authentication is performed using the user's private credentials.
+
+# Requirements
+
+To build and/or execute the program, the following requirements must be satisfied.
+
+## Java Version
+
+  - Java 11 or higher.
+
+## OCI Cloud Environment
+
+  - An account in the Oracle Cloud Infrastructure; either a free trial or a paid subscription.
+
+	https://docs.cloud.oracle.com/en-us/iaas/Content/GSG/Tasks/signingup.htm
+
+	https://www.oracle.com/cloud/free     (to create a free account)
+
+	https://docs.oracle.com/en-us/iaas/Content/GSG/Tasks/buysubscription.htm#upgrade_promotion   (to upgrade your free account to a paid account)
+
```

Comment:
I suggest to add here

- Cloud Shell VM comes with current versions of many useful tools and utilities pre-installed including Java and Maven. See [Managing Language Runtimes](https://docs.oracle.com/en-us/iaas/Content/API/Concepts/devcloudshellgettingstarted.htm#cloudshellgettingstarted_topic_managing_language_runtimes) to change Java version.